tests/mockito/junit4-agent/src/test/java/samples/powermockito/junit4/agent/MockFinalUsingAnnotationsTest.java

```diff
@@ -40,7 +40,7 @@
     private FinalDemo usingMockitoMockAnnotation;
 
     @SuppressWarnings("deprecation")
-    @org.mockito.MockitoAnnotations.Mock
+    @org.mockito.Mock
     private FinalDemo usingDeprecatedMockitoMockAnnotation;
```

Comment:
Thank you for notice, I think, it will be better to remove these fields to avoid misunderstanding in future.